docroot/themes/custom/uids_base/scss/global.scss

```diff
@@ -316,6 +316,7 @@ body:not(.page-node-type-article, .page-node-type-page).layout-builder-disabled
 
       .layout__region--second {
         order: 1;
+        margin-bottom: 2rem;
```

Comment:
Should `$mobile-height-gutter` be used instead? Paging @bspeare

Reply:
Yes, let's use that.